LayoutTests/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-request-resources.https-expected.txt

```diff
@@ -29,6 +29,10 @@ PASS Script load (url:https://localhost:9443/service-workers/service-worker/reso
 PASS Script load (url:https://localhost:9443/service-workers/service-worker/resources/sample?test27)
 PASS Script load (url:https://localhost:9443/service-workers/service-worker/resources/sample?test28)
 PASS Script load (url:https://localhost:9443/service-workers/service-worker/resources/sample?test29)
+PASS Module Script load (url:https://localhost:9443/service-workers/service-worker/resources/sample?test_module)
+FAIL Module Script load (url:https://localhost:9443/service-workers/service-worker/resources/sample?test_modulepreload) assert_equals: integrity of Module Script load (url:https://localhost:9443/service-workers/service-worker/resources/sample?test_modulepreload) must be sha384-foobar. expected "sha384-foobar" but got ""
+FAIL Module Script load (url:https://localhost:9443/service-workers/service-worker/resources/sample?test_moduleimport) assert_equals: integrity of Module Script load (url:https://localhost:9443/service-workers/service-worker/resources/sample?test_moduleimport) must be sha384-foobar. expected "sha384-foobar" but got ""
```

Comment:
This fails because of the way I implemented static import integrity (only on the response side)